### PR TITLE
(VDB-1279) LogSortedOffer + LogUnsortedOffer transformers

### PR DESCRIPTION
**Pull request by Gslaughl**
2 nearly-identical transformers in 2 separate commits

### CODE REVIEW

File: transformers/integration_tests/log_unsorted_offer_test.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		err = transformer.Execute(eventLogs)
+		Expect(err).NotTo(HaveOccurred())
+
+		var dbResult []logUnsortedOfferModel
+		err = db.Select(&dbResult, `SELECT offer_id FROM maker.log_unsorted_offer ORDER BY offer_id`)
+		Expect(err).NotTo(HaveOccurred())
+
+		Expect(len(dbResult)).To(Equal(1))
+		Expect(dbResult[0].OfferID).To(Equal("717050"))
+	})
```
Comment (rmulhol) [inline transformers/integration_tests/log_unsorted_offer_test.go:58]: Assuming we haven't seen any of these on the second contract?

File: transformers/integration_tests/log_unsorted_offer_test.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+})
+
+type logUnsortedOfferModel struct {
+	OfferID  string `db:"offer_id"`
+	HeaderID int64
+	LogID    int64 `db:"log_id"`
```
Comment (rmulhol) [inline transformers/integration_tests/log_unsorted_offer_test.go:64]: nbd but I think we can remove the `HeaderID` and `LogID` from this model if we're only querying for the `OfferID`. For that matter, we may not even need a model if we can just scan that value directly into a string or int64

File: transformers/test_data/log_sorted_offer.go [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/fakes"
+)
+
+var (
+	logSortedOfferOfferID, _ = new(big.Int).SetString("811648", 10)
```
Comment (rmulhol) [inline transformers/test_data/log_sorted_offer.go:17]: what's the motivation for putting this string into a big int? Seems like the only usage is calling `String()` on it?

File: transformers/test_data/log_sorted_offer.go [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	logSortedOfferModel = event.InsertionModel{
+		SchemaName: constants.MakerSchema,
+		TableName:  constants.LogSortedOfferTable,
+		OrderedColumns: []event.ColumnName{
+			event.HeaderFK, event.LogFK, event.AddressFK, constants.OfferId,
+		},
+		ColumnValues: event.ColumnValues{
+			event.HeaderFK:    LogSortedOfferEventLog.HeaderID,
+			event.LogFK:       LogSortedOfferEventLog.ID,
+			constants.OfferId: logSortedOfferOfferID.String(),
```
Comment (rmulhol) [inline transformers/test_data/log_sorted_offer.go:46]: this won't work 99% of the time but in this case we might be able to do `new(big.Int).SetBytes(logSortedOfferRawLog.Data).String()`

**Comment by rmulhol** [comment]
Paranoid me checking in to request a migration timestamp bump if #170 is merged first 😅

File: transformers/test_data/log_sorted_offer.go (hunk 3 above)
Comment (Gslaughl) [reply transformers/test_data/log_sorted_offer.go:17]: 🤦‍♂️

File: transformers/integration_tests/log_unsorted_offer_test.go (hunk 2 above)
Comment (Gslaughl) [reply transformers/integration_tests/log_unsorted_offer_test.go:64]: Good call!

File: transformers/integration_tests/log_unsorted_offer_test.go (hunk 1 above)
Comment (Gslaughl) [reply transformers/integration_tests/log_unsorted_offer_test.go:58]: correct 👍

**Pull request merged**